$NetBSD: patch-ac,v 1.1 2007/03/17 13:40:00 rillig Exp $

gcc4 complains:
numbers.c:44: error: static declaration of 'lusers_user_cnt' follows non-static declaration

--- include/toast.h.orig	1999-07-26 08:54:25.000000000 +0200
+++ include/toast.h	2007-03-17 14:37:27.000000000 +0100
@@ -26,7 +26,6 @@ extern	int	page_updown_key;
 extern	int	toast_action;
 
 extern	int	in_whois;
-extern	int	lusers_user_cnt;
 extern	int	bc_count;
 extern	char	*bc_chname;
 
